src/Model/ROI.py

```diff
@@ -953,6 +955,9 @@ def create_initial_rtss_from_ct(img_ds: pydicom.dataset.Dataset,
         raise ValueError(
             "The given dataset is missing a required tag 'StudyInstanceUID'")
 
+    if hasattr(rt_ss, "StudyID") is False or rt_ss.StudyID == "":
+        raise ValueError("The given dataset is missing a required tag 'StudyID'")
```

Comment:
technically, it's missing a tag that is required by OnkoDICOM.
The DICOM standard identifies StudyID as type 2 (attribute must be encoded, but an empty value is allowed), not type 1 (attribute must be encoded with a value).

Reply:
So this shouldn't throw an error.